src/components/recursive_tree/recursive_tree.tsx

```diff
@@ -211,27 +215,44 @@ export class EuiRecursiveTree extends Component<
       className
     );
 
+    const instructionsId = `${this.state.treeID}--instruction`;
+
     return (
       <EuiRecursiveTreeContext.Provider value={this.state.treeID}>
         <EuiText
           size={isCondensed ? 's' : 'm'}
           className="euiRecursiveTree__wrapper">
+          {!isNested && (
+            <EuiScreenReaderOnly>
+              <p id={instructionsId}>
+                {/* TODO: THIS NEEDS I18N */}
+                You can quickly navigate this list using arrow keys.
+              </p>
+            </EuiScreenReaderOnly>
+          )}
           <ul
             className={classes}
             id={this.state.treeID}
-            aria-describedby={this.state.treeID}
+            aria-describedby={!isNested ? instructionsId : undefined}
```

Comment:
Only giving the instructions on the top level so we don't annoy users with constant reminders